trueclass.rb

```diff
@@ -0,0 +1,32 @@
+class TrueClass
+  #
+  # call-seq:
+  #   true.to_s   ->  "true"
+  #
+  # The string representation of <code>true</code> is "true".
+  #
+  def to_s
+    "true".freeze
```

Comment:
This can be affected by the redefinition of `String#freeze`.
This should be `-"true"` or a frozen string literal.